app.py

```
@@ -360,7 +360,7 @@ def slot():
 @app.route("/productList")
 def product_list():
     # send pure JSON data
-    with open(f"{os.getcwd()}/DataBase/products.json", "r", encoding="utf-8") as file:
+    with open(f"{os.getcwd()}/database/products.json", "r", encoding="utf-8") as file:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)